README.rst

```diff
@@ -8,7 +8,7 @@ Welcome to ZGW Consumers' documentation!
 :Version: 0.26.2
 :Source: https://github.com/maykinmedia/zgw-consumers
 :Keywords: OpenAPI, Zaakgericht Werken, Common Ground, NLX
-:PythonVersion: 3.7, 3.8, 3.9
+:PythonVersion: 3.7, 3.8, 3.9, 3.10, 3.11
```

Comment:
I'm in favour of removing this alltogether - the `|python-versions|` badge already reflects this